backend/.env.local

```diff
@@ -1 +1,2 @@
-ENVIRONMENT=local
\ No newline at end of file
+ENVIRONMENT=local
+REACT_APP_STREAM_API_KEY=94ggnkahafdr
```

Comment:
I think we need a better strategy for our env vars. Having a backend/.env.local and a backend/.env is confusing. I recommend we do the following:
1. Move all .env.local into .env (so can delete .env.local) - note: this might require a change to [app.ts](https://github.com/Siyer2/uni-connector/blob/c41d4eff58389d4753854219e368797f7472b975/backend/src/app.ts#L9)
2. Create a .env.example that is NOT gitignored so people will know what's expected. It can look something like this:
```
REACT_APP_STREAM_API_SECRET=example
ENVIRONMENT=local
REACT_APP_STREAM_API_KEY=example
```
3. Message in general that you need this .env file in backend for the project to continue working

Also, I think it's a good idea to change `REACT_APP_STREAM_API_KEY` to `REACT_APP_STREAM_PUBLIC_KEY`

Reply:
where should I say that you need the .env file in the backend - in the README at the root of the project?